src/borg/testsuite/archiver.py

```diff
@@ -1298,9 +1298,12 @@ def test_list_size(self):
         self.cmd('init', self.repository_location)
         test_archive = self.repository_location + '::test'
         self.cmd('create', '-C', 'lz4', test_archive, 'input')
-        output = self.cmd('list', '--format', '{size} {csize} {path}{NL}', test_archive)
-        size, csize, path = output.split("\n")[1].split(" ")
+        output = self.cmd('list', '--format', '{size} {csize} {dsize} {dcsize} {path}{NL}', test_archive)
+        size, csize, dsize, dcsize, path = output.split("\n")[1].split(" ")
         assert int(csize) < int(size)
+        assert int(dcsize) < int(dsize)
+        assert int(dsize) <= int(size)
+        assert int(dcsize) <= int(csize)
```

Comment:
It does not fail either if I "cat video1.mp4 video2.mp4 > video-double.mp4". It calculates correctly the unique chunks (using identical random files of 160MiB)
```
                       Original size      Compressed size    Deduplicated size
This archive:              335.55 MB            335.55 MB            169.08 MB
All archives:              335.55 MB            335.55 MB            169.08 MB

                       Unique chunks         Total chunks
Chunk index:                      75                  145
```